webrender_api/src/display_item.rs

```diff
@@ -698,6 +700,22 @@ impl BorderRadius {
     }
 }
 
+impl Hash for BorderRadius {
```

Comment:
This shouldn't be needed now.

Reply:
If I remove the hash function, I will get the compilation errors:

313 |     pub top_left: LayoutSize,
       |     ^^^^^^^^^^^^^^^^^^^^^^^^ the trait `bitflags::<unnamed>::hash::Hash` is not implemented for `f32`

Or should I pass all radii to BoxShadowCacheKey?